### PR TITLE
cis: update azure pro  cis/usg tests

### PR DESCRIPTION
**Pull request by CalvoM**
## Proposed Commit Message

cis: update azure pro cis/usg enabling tests

Add tests for azure pro machines to test the enabling and disabling of usg/cis service.

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: features/ubuntu_pro.feature [hunk 1]
```diff
@@ -95,6 +95,22 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
             """
             <cis_or_usg>           +yes  +<cis-s>  +Security compliance and audit tools
             """
+        When I run `ua enable <cis_or_usg>` with sudo
```
Comment (lucasmoura) [inline features/ubuntu_pro.feature:98]: Why are we adding those extra cis/usg tests only for azure ?

**Comment by lucasmoura** [comment]
Tests are passing, but I think we can extend that to the other clouds as well

**Comment by orndorffgrant** [comment]
Test additions look good, I'll just run them all tomorrow before approving

**Review by orndorffgrant — APPROVED** [review]
Passed for me too! Merging

**Pull request merged**